Description: Fixes FTBFS against libav9
  This patch updates the API usage against libav
Author: Reinhard Tartler <siretart@ubuntu.com>
Origin: Ubuntu
Bug-Debian: http://bugs.debian.org/692192
Forwarded: no

diff --git a/examples/fpcalc.c b/examples/fpcalc.c
index 688a223..e5a4a19 100644
--- a/examples/fpcalc.c
+++ b/examples/fpcalc.c
@@ -42,7 +42,7 @@ int decode_audio_file(ChromaprintContext *chromaprint_ctx, int16_t *buffer1, int
 		goto done;
 	}
 
-	if (av_find_stream_info(format_ctx) < 0) {
+	if (avformat_find_stream_info(format_ctx, NULL) < 0) {
 		fprintf(stderr, "ERROR: couldn't find stream information in the file\n");
 		goto done;
 	}
@@ -65,7 +65,7 @@ int decode_audio_file(ChromaprintContext *chromaprint_ctx, int16_t *buffer1, int
 		goto done;
 	}
 
-	if (avcodec_open(codec_ctx, codec) < 0) {
+	if (avcodec_open2(codec_ctx, codec, NULL) < 0) {
 		fprintf(stderr, "ERROR: couldn't open the codec\n");
 		goto done;
 	}
diff --git a/src/fft_lib_avfft.h b/src/fft_lib_avfft.h
index 87c8034..a3e1d2d 100644
--- a/src/fft_lib_avfft.h
+++ b/src/fft_lib_avfft.h
@@ -25,6 +25,7 @@
 extern "C" {
 #include <libavcodec/avcodec.h>
 #include <libavcodec/avfft.h>
+#include <libavutil/mem.h>
 }
 #include "combined_buffer.h"
 
